aggregator.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"log"
+	"os"
+	"sync"
+	"time"
+)
+
+func getLatestOutput(instanceId string, sess *session.Session, withLatest bool) string {
```

Comment:
It would make it cleaner but it will require adding permissions (right now we could run this with only `getConsoleOutput` permission) and maybe even hold a list of Nitro instances.